.github/docker-images/rhel8-x64/Dockerfile

```diff
@@ -2,7 +2,17 @@ FROM registry.access.redhat.com/ubi8/ubi:latest
 
 SHELL ["/bin/bash", "-c"]
 
-RUN dnf install -y gcc gcc-c++ cmake wget python3 python3-pip git make sudo python3-devel
+RUN dnf install -y \
+    gcc \
+    gcc-c++ \
+    cmake \
+    wget \
+    git \
+    make \
+    sudo \
+    # RHEL8's default python3 is python3.6, which is EOL.
+    # So install python3.9 (latest version on this distro, circa Oct 2022)
+    python39-devel
```

Comment:
I think this should have python3-pip

Reply:
On this distro, you can install the one package `python39-devel` and it ends up installing the other python stuff like pip as a dependency. So I was just simplifying:
```
Installing:
 python39-devel              
Installing dependencies:
 libpkgconf                  
 pkgconf                     
 pkgconf-m4                  
 pkgconf-pkg-config          
 python39                    
 python39-libs               
 python39-pip-wheel          
 python39-setuptools-wheel   
Installing weak dependencies:
 python39-pip                
 python39-setuptools         
Enabling module streams:
 python39                    
```

In other distros you need to install them separately

If Linux variants all made a big effort to do everything the same ... you wouldn't have so many Linux variants